src/ctx.ts

```diff
@@ -0,0 +1,20 @@
+import * as vscode from 'vscode';
+import { SPMPackage } from './package';
+
+export class Ctx {
```

Comment:
Can we just name this type `Context`?

I know abbreviations are one of my pet peeves, but I very much prefer the clarity of a full name. Abbreviations like this noticeably increase my cognitive load when reading code.

Reply:
`Context` as with `Ctx` are both very vague. I copied this pattern from the rust vscode extension (they were using Ctx). Maybe rename to `SwiftContext` or `SPMContext`.

Reply:
OK. Perhaps we can go for `SwiftContext` and `SwiftPackage`?

Reply:
Done